src/3d/qgs3daxis.cpp

```diff
@@ -562,7 +631,7 @@ void Qgs3DAxis::populateMenu()
   {
     QWidget *mapCanvas = dynamic_cast<QWidget *>( eng->parent() );
     if ( mapCanvas == nullptr )
-      qDebug() << "NO CANVAS!";
+      qWarning() << "Qgs3DAxis: no canvas defined!";
```

Comment:
let's use QgsDebugMsg instead of qDebug / qWarning

Reply:
a missed replacement